en/docs/concepts/oidc/hybrid-client-profile.md

```diff
@@ -0,0 +1,242 @@
+
+# OpenID Connect Hybrid Flow
+
+WSO2 Identity Server supports the OpenID Connect hybrid flow for
+authentication.
+
+Specifying any of the following ` response_type ` values
+in an authorization request selects the hybrid flow for authentication:
+
+- [code token](#code-token)
+- [code id\_token](#code-id_token)
+- [code id\_token token](#code-id_token-token)
+
+To configure WSO2 Identity Server to support the OpenID Connect hybrid
+flow for authentication, you need to add following to the
+` <IS_HOME>/repository/conf/deployment.toml ` file.
+
+```toml
+[[oauth.custom_response_type]]
+name="code token"
+class="org.wso2.carbon.identity.oauth2.authz.handlers.HybridResponseTypeHandle"
+
+[[oauth.custom_response_type]]
+name="code id_token"
+class="org.wso2.carbon.identity.oauth2.authz.handlers.HybridResponseTypeHandler"
+
+[[oauth.custom_response_type]]
+name="code id_token token"
+class="org.wso2.carbon.identity.oauth2.authz.handlers.HybridResponseTypeHandler"
+```
+
+To understand how the ` response_type ` value specified
+in an authorization request selects the hybrid flow to be the
+authentication flow, let's take a look at the following
+` response_type ` values in detail .
+
+### code token
+
+This ` response_type ` requests a code and an access
+token from the authorization endpoint. 
+ 
+Following is a sample authorization request that uses
+` code token ` as the ` response_type ` :
```

Comment:
A sample authorization request that uses `code token` as the `response_type` is shown below.

Reply:
please follow the same sentence format when mentioning about samples shown below.